api_tests/lib/ledgers/ethereum.ts

```diff
@@ -0,0 +1,55 @@
+import { LedgerInstance } from "./ledger_runner";
+import { EthereumWallet } from "../wallets/ethereum";
+
+/**
+ * An instance of the Ethereum ledger for use in the e2e tests.
+ *
+ * This class is compatible with anything that implements {@link EthereumInstance}.
+ *
+ * Some of the e2e tests need an ERC20 token deployed to work properly.
+ * This class takes care of deploying such contract after the Ethereum
+ * blockchain is up and running.
+ *
+ * This class serves as an abstraction layer on top of Ethereum, regardless
+ * of which implementation is used (Docker container, parity, geth, etc).
+ */
+export default class EthereumLedger implements LedgerInstance {
+    public static async start(instance: EthereumInstance) {
+        await instance.start();
+
+        await instance.start();
```

Comment:
Duplicate line.

Reply:
Ah yeah that was an oversight that I fixed somewhere later :)

Gonna patch it out and force push!